app/services/invitation_service.rb

```diff
@@ -32,6 +32,14 @@ def self.invite_to_group(recipient_emails: nil,
     end
   end
 
+  def self.resend(invitation)
```

Comment:
I like it just because
```
.each { |invitation| resend invitation }
```

reads better than

```
.each { |invitation| resend invitation: invitation }
```